samples/msal-node-samples/ElectronSystemBrowserTestApp/src/Main.ts

```diff
@@ -104,6 +104,10 @@ export default class Main {
         Main.fetchManager = new FetchManager();
         Main.registerSubscriptions();
         Main.attemptSSOSilent();
+
+        if (process.env.automation != null) {
```

Comment:
can we check for the actual expected value of this instead?

Reply:
done